build-configs/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "build-configs",
+  "version": "0.0.1",
+  "description": "Build configs for the integreat project",
+  "main": "index.js",
+  "author": "Maximilian Ammann <max@maxammann.org>",
+  "license": "MIT",
+  "private": false
```

Comment:
Shouldn't private be true as this shouldn't be shared to npm

Reply:
Yes, probably, will look into it and change with one of the next PRs